### DISCUSSION TITLE
Erro no ficheiro, AuditFile.SourceDocuments.SalesInvoices.445Invoice.1Line.TaxExemptionReason está mal preenchido...

### DISCUSSION THREAD

**Discussion by AugCri** [assoft-portugal/SAF-T-AO#140] [General]
Bom dia, caríssimos.
ERRO: Erro no ficheiro, AuditFile.SourceDocuments.SalesInvoices.445Invoice.1Line.TaxExemptionReason

Alguém pode me dizer, por favor, como é que se preenche o "IVA - Regime Simplificado"?

Por qual motivo esse erro persiste, caríssimos? 
Já vi outros links aqui mas não me explicam algo diferente.
Por favor, alguém me pode dizer o que está errado? Alguém já passou por isso? Ou há um problema com o portal da AGT?

Muito obrigado
![IMAGEM_ERRO_SAFT_AO](https://user-images.githubusercontent.com/72359553/110142617-a59dfd00-7dd6-11eb-98f6-d29ed2a9782d.png)

**Comment by AugCri** [comment]
Ilustres, mas essa EXCEPÇÃO/ISENÇÃO (IVA - Regime Simplificado) realmente está a ser utilizada?

**Reply by cryptolopes** [reply]
feedback no [Telegram](https://t.me/saftao/4239)
recomendo ver #114

**Reply by AugCri** [reply]
Obrigado, ilustre @cryptolopes, pela sempre disponibilidade.

Na verdade, vim para comunicar a todos que possam ter a mesma dificuldade que aleatoriamente fui testando e descobri ontem que "Regime Simplificado" (sem o "IVA - "), FUNCIONA.

Eu me baseei no ficheiro, conforme pode ver na imagem mas, de certo, houve uma actualização que [esse](https://github.com/assoft-portugal/SAF-T-AO/blob/master/Resources/TaxExemptions.xml) ficheiro não acompanha.

Estamos com muitos problemas que não encontro explicação. Por exemplo, o portal não responde na maioria das vezes que fazemos um teste de um ficheiro, quando não encontra erro. Mas, de repente, emite a informação "Ficheiro validado" mas depois fica caladinho de novo! O que se deve a esse comportamento?

Resumindo, quanto à questão em epígrafe, o que funcionou, até agora que escrevo, é **Regime Simplificado**

Saudações

**Reply by AugCri** [reply]
Agora também já não funciona. Até ontem, estava tudo bem. Mas agora, o motivo "Regime Simplificado" também dá erro. Mas a que se deve essa instabilidade permanente, ilustres?

**Reply by cryptolopes** [reply]
Não faz sentido validar campos de texto livre, como o `TaxExemptionReason`. Ou melhor, não faz sentido impedir a submissão do ficheiro com base em validações de campos de texto livre. É a minha opinião.

**Comment by Adilson72** [comment]
Pessoal tive alguns problemas com o "IVA - Regime Simplificado" e "IVA - Regime de Exclusão"...
Depois de trocar o traço aí após o IVA "-" para este traço "–", o erro sumiu...


"IVA - Regime Simplificado" e "IVA - Regime de Exclusão" - Antes da alteração do traço
"IVA – Regime Simplificado" e "IVA – Regime de Exclusão" - Após alteração do traço
